operations/node_management/Add_Remove_Replace_NCNs/Update_NCN_BIOS_TPM_State.md

```diff
@@ -0,0 +1,86 @@
+# Update NCN BIOS TPM State
+
+## Description
+
+Enable Trusted Platform Module (TPM) in the BIOS on the new NCN if TPM is being used on the other NCNs.
+**Skip this step if TPM is not being used** and proceed to the next step to [Boot NCN and Configure](Boot_NCN.md).
+It is not required to disable TPM when it is not being used. Many types of hardware come with TPM enabled by default.
+
+## Current capabilities
+
+The following table lists the type of hardware where the TPM State can be enabled. Not all models from a given manufacturer support TPM.
+
+| **Manufacturer** | **Type**     |
+| ---------------- | ------------ |
+| Gigabyte         | `nodeBMC`    |
+| HPE              | `nodeBMC`    |
+
```

Comment:
Add a note about Intel NCNs/unsupported hardware, and skip this procedure if the NCN being added is Intel as SCSD does not support that hardware.